### ISSUE TITLE
[CI/CD Issues] app settings/configuration not getting applied - deployed app/UI doesn't come up

### ISSUE THREAD

**Issue by vhvb1989** [Azure/azure-dev#1033]
See original error here:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1947117&view=logs&j=98dfbbaa-4468-5ea8-0424-3e05dc9382fb&t=c52aa7c5-7457-5d87-592e-3a914f365cd6&l=216

Gate: TestTemplates Azure-Samples_todo-nodejs-mongo

```
2022-10-27T00:11:53.3744211Z ##[section]Starting: Test templates
2022-10-27T00:11:53.3757123Z ==============================================================================
2022-10-27T00:11:53.3758267Z Task         : Bash
2022-10-27T00:11:53.3759160Z Description  : Run a Bash script on macOS, Linux, or Windows
2022-10-27T00:11:53.3760227Z Version      : 3.201.1
2022-10-27T00:11:53.3760996Z Author       : Microsoft Corporation
2022-10-27T00:11:53.3762051Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
2022-10-27T00:11:53.3763120Z ==============================================================================
2022-10-27T00:11:53.5116462Z Generating script.
2022-10-27T00:11:53.5128941Z Formatted command: exec bash '/mnt/vss/_work/1/s/templates/tests/test-templates.sh' -e 'azd-template-test' -t 'Azure-Samples/todo-nodejs-mongo' -b 'staging' -s '1947117' -l 'eastus2' -p 'list'
2022-10-27T00:11:53.5138796Z ========================== Starting Command Output ===========================
2022-10-27T00:11:53.5151174Z [command]/usr/bin/bash /mnt/vss/_work/_temp/43ecbaf2-de88-4e5f-95ed-aa1bc432494a.sh
2022-10-27T00:11:53.5261771Z Creating new project folder @ '/home/cloudtest/azd-template-test-todo-nodejs-mongo-1947117'...
2022-10-27T00:11:53.5271187Z Initializing template 'Azure-Samples/todo-nodejs-mongo' with branch 'staging'
2022-10-27T00:11:53.7430652Z Initializing a new project in /home/cloudtest/azd-template-test-todo-nodejs-mongo-1947117
2022-10-27T00:11:53.7432846Z 
2022-10-27T00:11:53.7435005Z Downloading template |
2022-10-27T00:11:59.2627939Z Provisioning infrastructure for azd-template-test-todo-nodejs-mongo-1947117...
2022-10-27T00:12:04.7592059Z Planning infrastructure provisioning |
2022-10-27T00:12:04.7593976Z Generating Bicep parameters file... /
2022-10-27T00:12:04.7596168Z Compiling Bicep template... -
2022-10-27T00:12:09.4914474Z 
2022-10-27T00:12:09.4916738Z Infrastructure provisioning plan completed successfully
2022-10-27T00:12:09.4917991Z Provisioning Azure resources |
2022-10-27T00:12:12.7985158Z Provisioning Azure resources... /
2022-10-27T00:12:23.2583006Z Provisioning Azure resources can take some time.
2022-10-27T00:12:23.2584178Z 
2022-10-27T00:12:23.2585497Z You can view detailed progress in the Azure Portal:
2022-10-27T00:12:23.2590293Z https://portal.azure.com/#blade/HubsExtension/DeploymentDetailsBlade/overview/id/%2Fsubscriptions%2F***%2Fproviders%2FMicrosoft.Resources%2Fdeployments%2Fazd-template-test-todo-nodejs-mongo-1947117
2022-10-27T00:12:23.2591979Z 
2022-10-27T00:12:23.2594795Z Provisioning Azure resources... |
2022-10-27T00:12:26.5457984Z Created Resource group: rg-azd-template-test-todo-nodejs-mongo-1947117
2022-10-27T00:12:26.5459502Z Provisioning Azure resources (1 of ~4 completed)... |
2022-10-27T00:12:26.5460476Z Provisioning Azure resources (1 of ~4 completed)... /
2022-10-27T00:12:39.7062404Z Created App Service plan: plan-cwbvdpq4mslrm
2022-10-27T00:12:39.7064157Z Created Key vault: kv-cwbvdpq4mslrm
2022-10-27T00:12:39.7065239Z Provisioning Azure resources (3 of ~5 completed)... |
2022-10-27T00:12:39.7066320Z Provisioning Azure resources (3 of ~5 completed)... /
2022-10-27T00:12:52.9962737Z Created Log Analytics workspace: log-cwbvdpq4mslrm
2022-10-27T00:12:52.9963943Z Provisioning Azure resources (4 of ~5 completed)... |
2022-10-27T00:12:52.9964667Z Provisioning Azure resources (4 of ~5 completed)... /
2022-10-27T00:13:09.8472504Z Created Application Insights: appi-cwbvdpq4mslrm
2022-10-27T00:13:09.8490591Z Created Portal dashboard: dash-cwbvdpq4mslrm
2022-10-27T00:13:09.8491783Z Provisioning Azure resources (6 of ~8 completed)... |
2022-10-27T00:13:09.8492661Z Provisioning Azure resources (6 of ~8 completed)... /
2022-10-27T00:13:26.4836039Z Provisioning Azure resources (6 of ~8 completed)... -
2022-10-27T00:13:43.6469955Z Provisioning Azure resources (6 of ~8 completed)... \
2022-10-27T00:14:01.3567911Z Provisioning Azure resources (6 of ~8 completed)... |
2022-10-27T00:14:19.3701996Z Created Web App: app-web-cwbvdpq4mslrm
2022-10-27T00:14:19.3703708Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:14:19.3705635Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:14:37.1678890Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:14:54.7771236Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:15:12.4769907Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:15:30.3084219Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:15:48.0279951Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:16:05.7073895Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:16:23.6248481Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:16:41.3035983Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:16:59.0542500Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:17:16.7758970Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:17:34.4348150Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:17:52.5234332Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:18:10.5702285Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:18:28.5547492Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:18:46.6627061Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:19:04.6395476Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:19:22.6733844Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:19:40.6778756Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:19:58.5417396Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:20:16.2370745Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:20:33.9285872Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:20:52.8735654Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:21:10.6203283Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:21:28.2258109Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:21:45.9409288Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:22:03.7442836Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:22:21.5167880Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:22:39.2181417Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:22:56.8526201Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:23:14.4678495Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:23:39.4118398Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:23:56.1647273Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:24:14.0315996Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:24:32.7276038Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:24:50.4811114Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:25:08.3640773Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:25:26.0604113Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:25:43.7889148Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:26:01.4389576Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:26:19.1905144Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:26:36.8399464Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:26:54.5584085Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:27:12.3624273Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:27:30.1059378Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:27:47.7535029Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:28:05.4920177Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:28:23.2674480Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:28:41.0568069Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:28:59.6032826Z Provisioning Azure resources (9 of ~11 completed)... -
2022-10-27T00:29:17.2269635Z Provisioning Azure resources (9 of ~11 completed)... \
2022-10-27T00:29:34.8885419Z Provisioning Azure resources (9 of ~11 completed)... |
2022-10-27T00:29:52.6603908Z Provisioning Azure resources (9 of ~11 completed)... /
2022-10-27T00:30:10.3845426Z Created Azure Cosmos DB: cosmos-cwbvdpq4mslrm
2022-10-27T00:30:10.3846618Z Provisioning Azure resources (11 of ~12 completed)... |
2022-10-27T00:30:10.3847382Z Provisioning Azure resources (11 of ~12 completed)... /
2022-10-27T00:30:28.3231125Z Provisioning Azure resources (11 of ~13 completed)... -
2022-10-27T00:30:46.0535112Z Provisioning Azure resources (12 of ~15 completed)... \
2022-10-27T00:31:03.9074269Z Provisioning Azure resources (14 of ~15 completed)... |
2022-10-27T00:31:22.7003505Z Provisioning Azure resources (14 of ~15 completed)... /
2022-10-27T00:31:41.4879313Z Provisioning Azure resources (14 of ~15 completed)... -
2022-10-27T00:32:00.5718507Z Created Web App: app-api-cwbvdpq4mslrm
2022-10-27T00:32:00.5720551Z Provisioning Azure resources (16 of ~17 completed)... |
2022-10-27T00:32:19.4968282Z Provisioning Azure resources (17 of ~18 completed)... /
2022-10-27T00:32:38.8001321Z Provisioning Azure resources (18 of ~19 completed)... -
2022-10-27T00:32:58.1955187Z Provisioning Azure resources (19 of ~20 completed)... \
2022-10-27T00:32:58.1956111Z 
2022-10-27T00:32:58.1957093Z Azure resource provisioning completed successfully
2022-10-27T00:32:58.5206427Z View the resources created under the resource group rg-azd-template-test-todo-nodejs-mongo-1947117 in Azure Portal:
2022-10-27T00:32:58.5209684Z https://portal.azure.com/#@/resource/subscriptions/***/resourceGroups/rg-azd-template-test-todo-nodejs-mongo-1947117/overview
2022-10-27T00:32:58.5210743Z 
2022-10-27T00:32:58.5241360Z Deploying apps for azd-template-test-todo-nodejs-mongo-1947117...
2022-10-27T00:33:00.4996552Z Deploying service api...
2022-10-27T00:33:00.4998430Z Deploying service api... |
2022-10-27T00:33:00.5000257Z - Preparing packaging... /
2022-10-27T00:33:00.5001302Z - Installing dependencies... -
2022-10-27T00:33:17.1506285Z - Building service... \
2022-10-27T00:33:24.1871044Z - Copying deployment package... |
2022-10-27T00:33:24.1901526Z - Compressing deployment artifacts... /
2022-10-27T00:33:24.1904317Z - Compressing deployment artifacts... -
2022-10-27T00:33:24.2071221Z - Publishing deployment package... \
2022-10-27T00:33:59.2686582Z - Fetching endpoints for app service... |
2022-10-27T00:33:59.9430386Z Deployed service api
2022-10-27T00:33:59.9433758Z  - Endpoint: https://app-api-cwbvdpq4mslrm.azurewebsites.net/
2022-10-27T00:33:59.9434942Z 
2022-10-27T00:33:59.9436814Z Deploying service web...
2022-10-27T00:33:59.9438720Z Deploying service web... |
2022-10-27T00:33:59.9441006Z - Preparing packaging... /
2022-10-27T00:33:59.9443000Z - Installing dependencies... -
2022-10-27T00:34:18.3684492Z - Building service... \
2022-10-27T00:34:50.2421138Z - Copying deployment package... |
2022-10-27T00:34:50.2466320Z - Compressing deployment artifacts... /
2022-10-27T00:34:50.2468034Z - Compressing deployment artifacts... -
2022-10-27T00:34:50.3840979Z - Publishing deployment package... \
2022-10-27T00:35:13.6386457Z - Fetching endpoints for app service... |
2022-10-27T00:35:14.3312294Z Deployed service web
2022-10-27T00:35:14.3314417Z  - Endpoint: https://app-web-cwbvdpq4mslrm.azurewebsites.net/
2022-10-27T00:35:14.3315757Z 
2022-10-27T00:35:14.3337535Z Running template smoke tests for azd-template-test-todo-nodejs-mongo-1947117...
2022-10-27T00:35:15.1492519Z 
2022-10-27T00:35:15.1494896Z added 6 packages, and audited 7 packages in 478ms
2022-10-27T00:35:15.1495467Z 
2022-10-27T00:35:15.1496203Z found 0 vulnerabilities
2022-10-27T00:35:15.8666567Z Downloading Chromium 102.0.5005.40 (playwright build v1005)...
2022-10-27T00:35:15.9020751Z |                                                                                |   0% of 129.7 Mb
2022-10-27T00:35:16.0091153Z |■■■■■■■■                                                                        |  10% of 129.7 Mb
2022-10-27T00:35:16.0855363Z |■■■■■■■■■■■■■■■■                                                                |  20% of 129.7 Mb
2022-10-27T00:35:16.1585871Z |■■■■■■■■■■■■■■■■■■■■■■■■                                                        |  30% of 129.7 Mb
2022-10-27T00:35:16.2237845Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                                |  40% of 129.7 Mb
2022-10-27T00:35:16.2871955Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                        |  50% of 129.7 Mb
2022-10-27T00:35:16.3497979Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                |  60% of 129.7 Mb
2022-10-27T00:35:16.4078577Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                        |  70% of 129.7 Mb
2022-10-27T00:35:16.4729186Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                |  80% of 129.7 Mb
2022-10-27T00:35:16.5378937Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■        |  90% of 129.7 Mb
2022-10-27T00:35:16.6010975Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■| 100% of 129.7 Mb
2022-10-27T00:35:20.2513331Z Chromium 102.0.5005.40 (playwright build v1005) downloaded to /home/cloudtest/.cache/ms-playwright/chromium-1005
2022-10-27T00:35:20.2520101Z Downloading FFMPEG playwright build v1007...
2022-10-27T00:35:20.2724576Z |                                                                                |   0% of 2.6 Mb
2022-10-27T00:35:20.2793266Z |■■■■■■■■                                                                        |  10% of 2.6 Mb
2022-10-27T00:35:20.2824147Z |■■■■■■■■■■■■■■■■                                                                |  20% of 2.6 Mb
2022-10-27T00:35:20.2839400Z |■■■■■■■■■■■■■■■■■■■■■■■■                                                        |  30% of 2.6 Mb
2022-10-27T00:35:20.2851005Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                                |  40% of 2.6 Mb
2022-10-27T00:35:20.2868366Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                        |  50% of 2.6 Mb
2022-10-27T00:35:20.2885481Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                |  60% of 2.6 Mb
2022-10-27T00:35:20.2901455Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                        |  70% of 2.6 Mb
2022-10-27T00:35:20.2918198Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                |  80% of 2.6 Mb
2022-10-27T00:35:20.2932051Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■        |  90% of 2.6 Mb
2022-10-27T00:35:20.2949277Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■| 100% of 2.6 Mb
2022-10-27T00:35:20.3604627Z FFMPEG playwright build v1007 downloaded to /home/cloudtest/.cache/ms-playwright/ffmpeg-1007
2022-10-27T00:35:20.3606845Z Downloading Firefox 99.0.1 (playwright build v1323)...
2022-10-27T00:35:20.3796507Z |                                                                                |   0% of 76.6 Mb
2022-10-27T00:35:20.4278625Z |■■■■■■■■                                                                        |  10% of 76.6 Mb
2022-10-27T00:35:20.4656237Z |■■■■■■■■■■■■■■■■                                                                |  20% of 76.6 Mb
2022-10-27T00:35:20.5029821Z |■■■■■■■■■■■■■■■■■■■■■■■■                                                        |  30% of 76.6 Mb
2022-10-27T00:35:20.5426997Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                                |  40% of 76.6 Mb
2022-10-27T00:35:20.5801729Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                        |  50% of 76.6 Mb
2022-10-27T00:35:20.6177767Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                |  60% of 76.6 Mb
2022-10-27T00:35:20.6575520Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                        |  70% of 76.6 Mb
2022-10-27T00:35:20.6928887Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                |  80% of 76.6 Mb
2022-10-27T00:35:20.7260395Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■        |  90% of 76.6 Mb
2022-10-27T00:35:20.7633379Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■| 100% of 76.6 Mb
2022-10-27T00:35:22.7301850Z Firefox 99.0.1 (playwright build v1323) downloaded to /home/cloudtest/.cache/ms-playwright/firefox-1323
2022-10-27T00:35:22.7306875Z Downloading Webkit 15.4 (playwright build v1641)...
2022-10-27T00:35:22.7414898Z |                                                                                |   0% of 90.1 Mb
2022-10-27T00:35:22.8032883Z |■■■■■■■■                                                                        |  10% of 90.1 Mb
2022-10-27T00:35:22.8450103Z |■■■■■■■■■■■■■■■■                                                                |  20% of 90.1 Mb
2022-10-27T00:35:22.8846727Z |■■■■■■■■■■■■■■■■■■■■■■■■                                                        |  30% of 90.1 Mb
2022-10-27T00:35:22.9285065Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                                |  40% of 90.1 Mb
2022-10-27T00:35:22.9697108Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                        |  50% of 90.1 Mb
2022-10-27T00:35:23.0098989Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                                |  60% of 90.1 Mb
2022-10-27T00:35:23.0451341Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                        |  70% of 90.1 Mb
2022-10-27T00:35:23.0936172Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                |  80% of 90.1 Mb
2022-10-27T00:35:23.1318364Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■        |  90% of 90.1 Mb
2022-10-27T00:35:23.1757273Z |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■| 100% of 90.1 Mb
2022-10-27T00:35:25.3646773Z Webkit 15.4 (playwright build v1641) downloaded to /home/cloudtest/.cache/ms-playwright/webkit-1641
2022-10-27T00:35:26.3064573Z Loading env from: /home/cloudtest/azd-template-test-todo-nodejs-mongo-1947117/.azure/azd-template-test-todo-nodejs-mongo-1947117/.env
2022-10-27T00:35:26.3546165Z 
2022-10-27T00:35:26.3548249Z Running 1 test using 1 worker
2022-10-27T00:35:26.3548979Z 
2022-10-27T00:35:26.7891850Z baseUrl: https://app-web-cwbvdpq4mslrm.azurewebsites.net
2022-10-27T00:39:26.8666692Z   ✘  [chromium] › todo.spec.ts:4:1 › Create and delete item test (4m)
2022-10-27T00:39:27.3107929Z baseUrl: https://app-web-cwbvdpq4mslrm.azurewebsites.net
2022-10-27T00:43:28.6299446Z   ✘  [chromium] › todo.spec.ts:4:1 › Create and delete item test (retry #1) (4m)
2022-10-27T00:43:28.6425020Z 
2022-10-27T00:43:28.6430954Z 
2022-10-27T00:43:28.6461931Z   1) [chromium] › todo.spec.ts:4:1 › Create and delete item test ===================================
2022-10-27T00:43:28.6462889Z 
2022-10-27T00:43:28.6464351Z     [31mTimeout of 240000ms exceeded.[39m
2022-10-27T00:43:28.6464931Z 
2022-10-27T00:43:28.6466378Z     Error: [2mexpect([22m[31mreceived[39m[2m).[22mtoBeVisible[2m()[22m
2022-10-27T00:43:28.6467764Z     Call log:
2022-10-27T00:43:28.6469575Z       [2m- expect.toBeVisible with timeout 240000ms[22m
2022-10-27T00:43:28.6476606Z     [2m  - waiting for selector "text=My List >> nth=0"[22m
2022-10-27T00:43:28.6477462Z 
2022-10-27T00:43:28.6478078Z 
2022-10-27T00:43:28.6479610Z        5 |   await page.goto("/", { waitUntil: 'networkidle' });
2022-10-27T00:43:28.6480796Z        6 |
2022-10-27T00:43:28.6485236Z     >  7 |   await expect(page.locator("text=My List").first()).toBeVisible({
2022-10-27T00:43:28.6486604Z          |                                                      ^
2022-10-27T00:43:28.6487628Z        8 |     timeout: 240 * 1000,
2022-10-27T00:43:28.6488565Z        9 |   });
2022-10-27T00:43:28.6489395Z       10 |
2022-10-27T00:43:28.6489816Z 
2022-10-27T00:43:28.6491263Z         at /home/cloudtest/azd-template-test-todo-nodejs-mongo-1947117/tests/todo.spec.ts:7:54
2022-10-27T00:43:28.6491770Z 
2022-10-27T00:43:28.6492352Z     Pending operations:
2022-10-27T00:43:28.6493184Z       - expect.toBeVisible at todo.spec.ts:7:54
2022-10-27T00:43:28.6493553Z 
2022-10-27T00:43:28.6493816Z 
2022-10-27T00:43:28.6494681Z     Retry #1 ---------------------------------------------------------------------------------------
2022-10-27T00:43:28.6495104Z 
2022-10-27T00:43:28.6495851Z     [31mTimeout of 240000ms exceeded.[39m
2022-10-27T00:43:28.6496192Z 
2022-10-27T00:43:28.6497012Z     Error: [2mexpect([22m[31mreceived[39m[2m).[22mtoBeVisible[2m()[22m
2022-10-27T00:43:28.6497729Z     Call log:
2022-10-27T00:43:28.6498514Z       [2m- expect.toBeVisible with timeout 240000ms[22m
2022-10-27T00:43:28.6499419Z     [2m  - waiting for selector "text=My List >> nth=0"[22m
2022-10-27T00:43:28.6499774Z 
2022-10-27T00:43:28.6500009Z 
2022-10-27T00:43:28.6500784Z        5 |   await page.goto("/", { waitUntil: 'networkidle' });
2022-10-27T00:43:28.6501851Z        6 |
2022-10-27T00:43:28.6502531Z     >  7 |   await expect(page.locator("text=My List").first()).toBeVisible({
2022-10-27T00:43:28.6503254Z          |                                                      ^
2022-10-27T00:43:28.6503990Z        8 |     timeout: 240 * 1000,
2022-10-27T00:43:28.6504592Z        9 |   });
2022-10-27T00:43:28.6505142Z       10 |
2022-10-27T00:43:28.6505396Z 
2022-10-27T00:43:28.6506316Z         at /home/cloudtest/azd-template-test-todo-nodejs-mongo-1947117/tests/todo.spec.ts:7:54
2022-10-27T00:43:28.6506786Z 
2022-10-27T00:43:28.6507340Z     Pending operations:
2022-10-27T00:43:28.6508167Z       - expect.toBeVisible at todo.spec.ts:7:54
2022-10-27T00:43:28.6508532Z 
2022-10-27T00:43:28.6509303Z 
2022-10-27T00:43:28.6510819Z     attachment #1: trace (application/zip) ---------------------------------------------------------
2022-10-27T00:43:28.6512005Z     test-results/todo-Create-and-delete-item-test-chromium-retry1/trace.zip
2022-10-27T00:43:28.6512723Z     Usage:
2022-10-27T00:43:28.6513007Z 
2022-10-27T00:43:28.6513919Z         npx playwright show-trace test-results/todo-Create-and-delete-item-test-chromium-retry1/trace.zip
2022-10-27T00:43:28.6514373Z 
2022-10-27T00:43:28.6515252Z     ------------------------------------------------------------------------------------------------
2022-10-27T00:43:28.6515679Z 
2022-10-27T00:43:28.6516431Z   Slow test file: [chromium] › todo.spec.ts (8m)
2022-10-27T00:43:28.6517219Z   Consider splitting slow test files to speed up parallel execution
2022-10-27T00:43:28.6517597Z 
2022-10-27T00:43:28.6518121Z   1 failed
2022-10-27T00:43:28.6519061Z     [chromium] › todo.spec.ts:4:1 › Create and delete item test ====================================
2022-10-27T00:43:28.6674260Z ##[error]Bash exited with code '1'.
2022-10-27T00:43:28.6689641Z ##[section]Finishing: Test templates
```

**Comment by vhvb1989** [comment]
One more here:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1943999&view=logs&j=9392ca66-1ee4-52af-022d-b22076a99114&t=6db148d5-c790-543c-3e98-57830bb8eafe&l=169

Gate: TestTemplates Azure-Samples_todo-python-mongo

**Comment by vhvb1989** [comment]
One more here:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1943999&view=logs&j=255759e9-95fd-5b22-cada-8803e798a082&t=e39b3856-d7f9-53f8-aecf-19f4626fff0c&l=159

Gate: TestTemplates Azure-Samples_todo-csharp-sql

**Comment by vhvb1989** [comment]
One more:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1941119&view=logs&j=98dfbbaa-4468-5ea8-0424-3e05dc9382fb&s=96ac2280-8cb4-5df5-99de-dd2da759617d&t=c52aa7c5-7457-5d87-592e-3a914f365cd6&l=183

Gate: TestTemplates Azure-Samples_todo-nodejs-mongo

**Comment by vhvb1989** [comment]
One more:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1947117&view=logs&j=98dfbbaa-4468-5ea8-0424-3e05dc9382fb&t=c52aa7c5-7457-5d87-592e-3a914f365cd6&l=216

Gate: TestTemplates Azure-Samples_todo-nodejs-mongo

**Comment by weikanglim** [comment]
Confirmed that this is indeed the appsettings issue we've seen in previous versions of `azd` as well:

Example app service with no configuration even though Azure resource deployment reported successful:
https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/2cd617ea-1866-46b1-90e3-fffb087ebf9b/resourceGroups/rg-AZD-TEMPLATE-UPPER-TEST-todo-python-mongo-1956344/providers/Microsoft.Web/sites/app-api-tjfoh357mbgpa/configuration

**Comment by rajeshkamal5050** [comment]
Removing blocker given that we had two successful runs of template test pipelines
- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1958443&view=results
- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1957561&view=results

We can monitor for some more time and close it if they become stable.

**Comment by weikanglim** [comment]
Fixed by #996

Four more successful runs:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1958788&view=results
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1959777&view=results
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1959819&view=results (failed due different issue -- KeyVault)
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1960666&view=results

Closing this
